### PR TITLE
Update to Fairy Stockfish largeboards 14.1

### PR DESCRIPTION
**Pull request by TheYoBots**
Related to https://github.com/WandererXII/shoginet/pull/13

### CODE REVIEW

**Comment by sukrosono** [comment]
would be happy to see this merge to the base branch.

**Comment by ddugovic** [comment]
Yaneura V7.00beta is released.

**Comment by SriMethan** [comment]
YaneuraOu V7.00beta 2 has also been released

**Comment by TheYoBots** [comment]
@WandererXII please provide your review on this Pull Request. Thanks!

File: app/templating/AiHelper.scala [hunk 1]
```diff
@@ -8,7 +8,7 @@ import lila.app.ui.ScalatagsTemplate._
 trait AiHelper { self: I18nHelper =>
 
   def aiName(level: Int, withRating: Boolean = true)(implicit lang: Lang): String = {
-    val name = trans.aiNameLevelAiLevel.txt("Engine", level)
+    val name = trans.aiNameLevelAiLevel.txt("YaneuraOu V7.00", level)
```
Comment (ddugovic) [inline app/templating/AiHelper.scala:11]: I'm not sure that names or version numbers usually go in translation contexts.

Comment (TheYoBots) [reply app/templating/AiHelper.scala:11]: On crowdin `aiNameLevelAiLevel` is translated as `%1$s level %2$s` where `%1$s` will be `YaneuraOu V7.00`. So I don't think this should be a problem right

**Comment by WandererXII** [comment]
Sorry for the delay.
For minishogi and games from position Fairy-Stockfish is used, is that reflected here?
I guess that should be addressed in several places, because currently it is not clear whether you are playing against fairy or yaneuraou.

**Comment by TheYoBots** [comment]
> Sorry for the delay. For minishogi and games from position Fairy-Stockfish is used, is that reflected here? I guess that should be addressed in several places, because currently it is not clear whether you are playing against fairy or yaneuraou.

@WandererXII I though about that too, but shoginet doesn't use Fairy Stockfish anywhere so I wasn't sure how the AI even plays minishogi and From Position games. 
I also think this might be the reason for https://github.com/WandererXII/lishogi/issues/496

**Comment by WandererXII** [comment]
> > Sorry for the delay. For minishogi and games from position Fairy-Stockfish is used, is that reflected here? I guess that should be addressed in several places, because currently it is not clear whether you are playing against fairy or yaneuraou.
> 
> @WandererXII I though about that too, but shoginet doesn't use Fairy Stockfish anywhere so I wasn't sure how the AI even plays minishogi and From Position games. I also think this might be the reason for #496

I just use tweaked shoginet where yaneuraOu is replaced with Fairy-Stockfish. For #496 I will try updating Fairy, since I still use some older version and set some timeouts in case it hangs.

**Comment by TheYoBots** [comment]
@WandererXII I've updated this pull request and my pull request to shoginet to support Fairy Stockfish and to support minishogi. Currently I am facing a few errors with https://github.com/WandererXII/shoginet/pull/13 , but I'm trying a fix.

If you want anything else to be changed, please do mention it. Thanks!

**Pull request closed without merging**